src/backends/kafka/mappings.rs

```diff
@@ -0,0 +1,20 @@
+use crate::types::errors::KafkaErrorCode;
```

Comment:
nit: Mappings is not a very descriptive name. How about renaming this `errors.rs` and also moving `KafkaErrorCode` into here?